### PR TITLE
Fix users feedback

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2006

### CODE REVIEW

File: frontend/components/EDPBanner.vue [hunk 1]
```diff
@@ -3,51 +3,6 @@
     <div v-if="loading" class="text-center py-4">
       <Loading />
     </div>
-    <div v-else>
```
Comment (AndreiNavoichyk) [inline frontend/components/EDPBanner.vue:6]: We don't need the EDP banner at all. So just remove the whole component and its usage.

File: frontend/components/Ranking.vue [hunk 2]
```diff
@@ -155,19 +155,19 @@
...
                   v-b-tooltip.hover
-                  class="select"
-                  title="Select / Unselect validator"
                   @click="toggleSelected(data.item.stashAddress)"
                 >
                   <font-awesome-icon
                     v-if="data.item.selected"
                     icon="hand-paper"
                     class="selected text-selected"
+                    :title="$t('pages.accounts.remove_from_favorites')"
```
Comment (AndreiNavoichyk) [inline frontend/components/Ranking.vue:165]: Should the icon be here `hand-paper`?

File: frontend/components/Ranking.vue [hunk 3]
```diff
@@ -155,19 +155,19 @@
...
                   />
                   <font-awesome-icon
                     v-else
                     icon="hand-paper"
                     class="unselected text-secondary"
+                    :title="$t('pages.accounts.add_to_favorites')"
```
Comment (AndreiNavoichyk) [inline frontend/components/Ranking.vue:172]: Should the icon be here `hand-paper`?

File: frontend/components/Ranking.vue [hunk 4]
```diff
@@ -342,7 +343,7 @@ export default {
         },
         {
           key: 'selected',
-          label: 'Select',
+          label: 'Favourites',
```
Comment (AndreiNavoichyk) [inline frontend/components/Ranking.vue]: Let's use the single variant (without `u`) for `Favorite` spelling everywhere.

File: frontend/components/ReceivedTransfers.vue [hunk 5]
```diff
@@ -170,6 +181,16 @@ export default {
           label: 'Amount',
           sortable: true,
         },
+        {
```
Comment (AndreiNavoichyk) [inline frontend/components/ReceivedTransfers.vue]: Let's not show it in the table. We need these fields only for CSV export.

File: frontend/components/ReceivedTransfers.vue [hunk 6]
```diff
@@ -221,7 +243,11 @@ export default {
               block_number: event.block_number,
               from: JSON.parse(event.data)[0],
               to: this.accountId,
-              amount: JSON.parse(event.data)[2],
+              amount: new BigNumber(JSON.parse(event.data)[2])
```
Comment (AndreiNavoichyk) [inline frontend/components/ReceivedTransfers.vue]: Move this logic to utils or smth and reuse.

File: frontend/components/ReceivedTransfers.vue [hunk 7]
```diff
@@ -221,7 +243,11 @@ export default {
...
+                .div(new BigNumber(10).pow(network.tokenDecimals))
```
Comment (AndreiNavoichyk) [inline frontend/components/ReceivedTransfers.vue]: Please, calculate `new BigNumber(10).pow(*)` once

File: frontend/components/ReceivedTransfers.vue [hunk 8]
```diff
@@ -221,7 +243,11 @@ export default {
...
+                .toNumber(),
+              timestamp: event.timestamp,
+              utc_time: new Date(event.timestamp * 1000).toUTCString(),
```
Comment (AndreiNavoichyk) [inline frontend/components/ReceivedTransfers.vue]: Move this logic to utils or smth and reuse

File: frontend/frontend.config.js [hunk 9]
```diff
@@ -1,18 +1,18 @@
 export const network = {
```
Comment (AndreiNavoichyk) [inline frontend/frontend.config.js]: revert back

File: frontend/components/Ranking.vue (hunk 2 above)
Comment (DmitriyMolch) [reply frontend/components/Ranking.vue:165]: That was for mobile devices. We need to check it in the view of the mobile device.

File: frontend/components/Ranking.vue (hunk 3 above)
Comment (DmitriyMolch) [reply frontend/components/Ranking.vue:172]: Is should be a star icon for mobile devices

File: frontend/components/Ranking.vue (hunk 4 above)
Comment (DmitriyMolch) [reply frontend/components/Ranking.vue]: fixed

File: frontend/components/EDPBanner.vue (hunk 1 above)
Comment (DmitriyMolch) [reply frontend/components/EDPBanner.vue:6]: fixed

File: frontend/components/ReceivedTransfers.vue (hunk 7 above)
Comment (DmitriyMolch) [reply frontend/components/ReceivedTransfers.vue]: fixed

File: frontend/components/ReceivedTransfers.vue (hunk 8 above)
Comment (DmitriyMolch) [reply frontend/components/ReceivedTransfers.vue]: fixed

File: frontend/frontend.config.js (hunk 9 above)
Comment (DmitriyMolch) [reply frontend/frontend.config.js]: fixed

**Review by AndreiNavoichyk — APPROVED** [review]

File: frontend/components/ReceivedTransfers.vue [hunk 10]
```diff
@@ -19,7 +19,14 @@
         </b-col>
       </b-row>
       <JsonCSV
-        :data="transfers"
+        :data="
+          transfers.map((transfer) => {
+            const cloned = { ...transfer }
+            cloned.amount = formatAmountToDecimal(transfer.amount)
+            cloned['utc_time'] = getDateFromTimestamp(transfer.timestamp)
```
Comment (AndreiNavoichyk) [inline frontend/components/ReceivedTransfers.vue]: Let's use `date_time` header

File: frontend/components/SentTransfers.vue [hunk 11]
```diff
@@ -19,7 +19,14 @@
         </b-col>
       </b-row>
       <JsonCSV
-        :data="transfers"
+        :data="
+          transfers.map((transfer) => {
+            const cloned = { ...transfer }
+            cloned.amount = formatAmountToDecimal(transfer.amount)
+            cloned['utc_time'] = getDateFromTimestamp(transfer.timestamp)
```
Comment (AndreiNavoichyk) [inline frontend/components/SentTransfers.vue]: Let's use `date_time` header.

File: frontend/components/ReceivedTransfers.vue (hunk 10 above)
Comment (DmitriyMolch) [reply frontend/components/ReceivedTransfers.vue]: fixed

File: frontend/components/SentTransfers.vue (hunk 11 above)
Comment (DmitriyMolch) [reply frontend/components/SentTransfers.vue]: fixed

File: frontend/CHANGELOG.md [hunk 12]
```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## vNext
```
Comment (AndreiNavoichyk) [inline frontend/CHANGELOG.md:3]: Please, move the content to this [CHANGELOG](https://github.com/Cerebellum-Network/cerestats/blob/dev-cere/CHANGELOG.md) and add the [FE] prefix before each item.

File: frontend/CHANGELOG.md [hunk 13]
```diff
@@ -1,7 +1,10 @@
...
-
+- Removed EDP banner
+- Updated sent transfers, received transfers excell format
```
Comment (AndreiNavoichyk) [inline frontend/CHANGELOG.md]: It is not only format, you added new columns. Please, update.

File: frontend/CHANGELOG.md (hunk 12 above)
Comment (DmitriyMolch) [reply frontend/CHANGELOG.md:3]: fixed

File: frontend/CHANGELOG.md (hunk 13 above)
Comment (DmitriyMolch) [reply frontend/CHANGELOG.md]: fixed

**Pull request merged**